starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Consider starting your css reseting some styles. That way you reduce browser inconsistencies. We will see more advanced stuff, for now, you can do:
```
/* ---- reset ---- */
h1, body, html, main {
  margin: 0;
}
```